polars/polars-ops/src/series/ops/cut.rs

```diff
@@ -0,0 +1,108 @@
+use std::iter::once;
+
+use polars_core::prelude::*;
+
+use crate::series::ops::SeriesSealed;
+
+pub trait CutSeries: SeriesSealed {
```

Comment:
We only dispatch via expression. Can make it simpler by just having a function that accepts a `&Series` and arguments?